src/VisualStudio/Core/Def/ServicesVisualStudio.csproj

```diff
@@ -19,6 +19,9 @@
   </PropertyGroup>
   <ItemGroup Label="Build Items">
     <Compile Include="..\..\..\Compilers\Core\Desktop\IVsSQM.cs" />
+    <Compile Include="..\..\..\Compilers\Helpers\ShadowCopyAnalyzerAssemblyLoader.cs">
+      <Link>InternalUtilities\ShadowCopyAnalyzerAssemblyLoader.cs</Link>
```

Comment:
Is this just to prevent this type from being public? I thought we used linked files only for *true* data structure utilities that need to be shared across compilers/workspace/VS layers, not for actual functional components.
Another option would be to expose an AnalyzerAssemblyLoaderProvider service in the core compiler layer that exposes statically created instances of both kinds of loaders as IAnalyzerAssemblyLoader, however that does add to our public surface area.